app/controllers/pages_controller.rb

```diff
@@ -4,6 +4,6 @@ class PagesController < ApplicationController
   skip_before_action :disable_caching
 
   def page_not_found
-    render layout: false, status: 404
+    render layout: false, status: 404, formats: %i[css html png]
```

Comment:
Ah, I see below that you removed the constraint matching from the route, and expanded it to include `css` and `png`. 

Why wouldn't we want to include other formats like `js` and `jpg`?